### ISSUE TITLE
failed to handle multiple devices for container. Skipping Filesystem stats

### ISSUE THREAD

**Issue by PI-Victor** [openshift/origin#11548]
keeps spamming origin logs. not sure if it's trivial or serious, so i'm making this p3.

![screen shot 2016-10-25 at 02 19 27](https://cloud.githubusercontent.com/assets/6294120/19668664/9342de88-9a59-11e6-812e-4f4e0aea3ebb.png)

```
I1025 03:17:27.740747    3265 conversion.go:133] failed to handle multiple devices for container. Skipping Filesystem stats
I1025 03:17:27.740782    3265 conversion.go:133] failed to handle multiple devices for container. Skipping Filesystem stats
I1025 03:17:37.765395    3265 conversion.go:133] failed to handle multiple devices for container. Skipping Filesystem stats
I1025 03:17:37.765418    3265 conversion.go:133] failed to handle multiple devices for container. Skipping Filesystem stats
I1025 03:17:47.789968    3265 conversion.go:133] failed to handle multiple devices for container. Skipping Filesystem stats
I1025 03:17:47.789995    3265 conversion.go:133] failed to handle multiple devices for container. Skipping Filesystem stats
```
##### Version

[ipalade@openshift-devlab ~]$ openshift version
openshift v1.4.0-alpha.0+524f0f6-747-dirty
kubernetes v1.4.0+776c994
etcd 3.1.0-alpha.1
[ipalade@openshift-devlab ~]$ oc version
oc v1.4.0-alpha.0+524f0f6-747-dirty
kubernetes v1.4.0+776c994
features: Basic-Auth

Server https://127.0.0.1:8443
openshift v1.4.0-alpha.0+524f0f6-747-dirty
kubernetes v1.4.0+776c994
##### Steps To Reproduce

appears in the server logs when starting the server

@mfojtik i saw conversion.go and thought of you, have any idea who to get involved or if it's non-trivial?

**Comment by PI-Victor** [comment]
also, this shows up in loglevel=2 it seems it doesn't show up in normal server logs

**Comment by jperville** [comment]
Having this issue with the newly released origin v1.4.1 CentOS packages. The issue reproduces in IshentRas/cookbook-openshift3#60 , just run `journalctl -f -u origin-node.service` from inside the kitchen VM.

**Comment by derekwaynecarr** [comment]
its spam, we have a fix in progress here: https://github.com/google/cadvisor/pull/1577
